cdap-docs/application-templates/source/etl/templates/transformations/projection.rst

```diff
@@ -6,19 +6,35 @@
 Transformations: Projection
 ===========================
 
-.. rubric:: Description: Projection Transform: Drop, Rename, and Cast Fields
+.. rubric:: Description
 
-Projection transform that lets you drop, rename, and cast fields to a different type.
+A transform that can drop fields, rename fields, and cast fields to a different type.
+Operations are performed in a single step and are not applied in order on top of each other.
```

Comment:
the point this is trying to make is that there is no order, no intermediate state. everything is applied only to the input